src/SkprConfig.php

```diff
@@ -79,14 +79,38 @@ public function load(string $filename = self::CONFIG_FILENAME): self {
         return $this;
       }
       $this->config = json_decode($data, TRUE);
-      array_walk($this->config, function ($value, $key) {
-        // Store as env vars.
-        putenv($this->convertToEnvVarName($key) . '=' . $value);
-      });
     }
     return $this;
   }
 
+  /**
+   * Puts all Skpr config into env vars.
+   *
+   * @param string[] $include
+   *   (optional) The list of config keys to include. An empty array will
+   *   include all keys.
+   */
+  public function putEnvs(array $include = []) {
+    array_walk($this->config, function ($value, $key) use ($include) {
+      // Store as env vars.
+      if (empty($include) || in_array($key, $include, TRUE)) {
+        $this->putenv($key, $value);
+      }
+    });
+  }
+
+  /**
+   * Puts a Skpr config into an env var.
+   *
+   * @param string $key
+   *   The config key.
+   * @param string $value
+   *   The config value.
+   */
+  public function putEnv($key, $value) {
```

Comment:
I find this one a little be weird, because the value should come from `$this->config`? But I can see how it came about. No suggestions/recommendations.
Should `$skpr->putEnv('my.config')` work?

Reply:
Ah, fair point.